docs/examples/basic.dto.xml

```diff
@@ -12,6 +12,7 @@
 		<field name="attributes" type="string[]" />
 		<field name="manufactured" type="\Cake\I18n\FrozenDate" />
 		<field name="owner" type="Owner"/>
+		<field name="service" type="string[]|int|float"/>
```

Comment:
looks good
what about array with different types?

    string|int|float[]

how could this be done? whats the valid phpdoc for it?

Reply:
OK

Reply:
I guess it's not official yet:
- https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#appendix-a-types
- https://youtrack.jetbrains.com/issue/WI-20193